lstchain/tools/lstchain_create_irf_files.py

```diff
@@ -0,0 +1,414 @@
+    name = "IRFFITSWriter"
+    description = __doc__
+    example = """
+    To generate IRFs from MC gamma only, using default cuts/binning:
+    > lstchain_create_irf_files
+        -g /path/to/DL2_MC_gamma_file.h5
+        -o /path/to/irf.fits.gz
+        --point-like (Only for point_like IRFs)
+        --overwrite
+
+    Or to generate all 4 IRFs, using default cuts/binning:
+    > lstchain_create_irf_files
+        -g /path/to/DL2_MC_gamma_file.h5
+        -p /path/to/DL2_MC_proton_file.h5
+        -e /path/to/DL2_MC_electron_file.h5
+        -o /path/to/irf.fits.gz
+        --point-like (Only for point_like IRFs)
+
+    Or use a config file for cuts and binning information:
+    > lstchain_create_irf_files
+        -g /path/to/DL2_MC_gamma_file.h5
+        -o /path/to/irf.fits.gz
+        --point-like (Only for point_like IRFs)
+        --config /path/to/config.json
+
+    Or pass the selection cuts from command-line:
+    > lstchain_create_irf_files
+        -g /path/to/DL2_MC_gamma_file.h5
+        -o /path/to/irf.fits.gz
+        --point-like (Only for point_like IRFs)
+        --fixed-gh-cut 0.9
+        --fixed-theta-cut 0.2
+        --irf-obs-time 50
+    """
+
+    input_gamma_dl2 = traits.Path(
+        help="Input MC gamma DL2 file",
+        exists=True,
+        directory_ok=False,
+        file_ok=True
+    ).tag(config=True)
+
+    input_proton_dl2 = traits.Path(
+        help="Input MC proton DL2 file",
+        exists=True,
+        directory_ok=False,
+        file_ok=True
+    ).tag(config=True)
+
+    input_electron_dl2 = traits.Path(
+        help="Input MC electron DL2 file",
+        exists=True,
+        directory_ok=False,
+        file_ok=True
+    ).tag(config=True)
+
+    output_irf_file = traits.Path(
+        help="IRF output file",
+        directory_ok=False,
+        file_ok=True,
+        default_value="./irf.fits.gz",
+    ).tag(config=True)
+
+    irf_obs_time = traits.Float(
+        help="Observation time for IRF in hours",
+        default_value=50,
+    ).tag(config=True)
+
+    point_like = traits.Bool(
+        help="True for point_like IRF, False for Full Enclosure",
+        default_value=False,
+    ).tag(config=True)
+
+    overwrite = traits.Bool(
+        help="If True, overwrites existing output file without asking",
+        default_value=False,
+    ).tag(config=True)
+
+    classes = [EventSelector, DL3FixedCuts, DataBinning]
+
+    aliases = {
+        ("g", "input-gamma-dl2"): "IRFFITSWriter.input_gamma_dl2",
+        ("p", "input-proton-dl2"): "IRFFITSWriter.input_proton_dl2",
+        ("e", "input-electron-dl2"): "IRFFITSWriter.input_electron_dl2",
+        ("o", "output-irf-file"): "IRFFITSWriter.output_irf_file",
+        "irf-obs-time": "IRFFITSWriter.irf_obs_time",
+        "fixed-gh-cut": "DL3FixedCuts.fixed_gh_cut",
+        "fixed-theta-cut": "DL3FixedCuts.fixed_theta_cut",
+        "allowed-tels": "DL3FixedCuts.allowed_tels",
+        "config": "EventSelector.config",
```

Comment:
This overwrites the default alias for the config file, this needs to be removed.